cpp/src/rolling/rolling.cu

```diff
@@ -410,48 +410,14 @@ struct rolling_window_launcher {
                                                DeviceMax::template identity<T>(),
                                                stream);
     } else {
-      CUDF_EXPECTS(op == aggregation::COUNT_VALID || op == aggregation::COUNT_ALL,
-                   "COUNT_VALID or COUNT_ALL aggregation only is expected");
-      size_type valid_count;
-      if (op == aggregation::COUNT_ALL)
-        valid_count = kernel_launcher<T,
-                                      DeviceCount,
-                                      aggregation::COUNT_ALL,
-                                      PrecedingWindowIterator,
-                                      FollowingWindowIterator>(input,
-                                                               output_view,
-                                                               preceding_window_begin,
-                                                               following_window_begin,
-                                                               min_periods,
-                                                               agg,
-                                                               string_view{},
-                                                               stream);
-      else
-        valid_count = kernel_launcher<T,
-                                      DeviceCount,
-                                      aggregation::COUNT_VALID,
-                                      PrecedingWindowIterator,
-                                      FollowingWindowIterator>(input,
-                                                               output_view,
-                                                               preceding_window_begin,
-                                                               following_window_begin,
-                                                               min_periods,
-                                                               agg,
-                                                               string_view{},
-                                                               stream);
-      output->set_null_count(output->size() - valid_count);
+      CUDF_FAIL("MIN and MAX are the only supported aggregation types here");
```

Comment:
What happened to COUNT_VALID and COUNT_ALL for strings?

Reply:
The changes I made to the traits allowed these to just be handled in the default implementation instead of having to be duplicated within the string specialization.  See 
https://github.com/rapidsai/cudf/pull/4972#discussion_r420262528